feconf.py

```diff
@@ -599,12 +599,13 @@ def get_empty_ratings():
 
 # Event types.
 EVENT_TYPE_STATE_HIT = 'state_hit'
+EVENT_TYPE_STATE_FINISH = 'state_finish'
```

Comment:
Should be STATE_FINISHED for consistency? (Past tense.)

Reply:
Done.